bloom/generators/debian/templates/rules.em

```diff
@@ -59,4 +60,7 @@ override_dh_auto_install:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
-	dh_auto_install
+	dh_auto_install --destdir=$(CURDIR)/debian/@(Package)
```

Comment:
From my experience as debian maintainer, the destdir is necessary when building -dbg packages, probably because we don't have a single package in the control file any more.

In debian I use `--destdir` with a relative path `debian/$(Package)`, which is the form that appears in the [documentation](https://wiki.debian.org/DebugPackage). @nayomal was the CURDIR strictly necessary for the integration with bloom?